src/adaptermanager.js

```diff
@@ -246,7 +246,9 @@ exports.enableAnalytics = function (config) {
 };
 
 exports.setBidderSequence = function (order) {
-  _bidderSequence = order;
+  if (order === CONSTANTS.ORDER.RANDOM) {
```

Comment:
looks like it was moved from https://github.com/prebid/Prebid.js/pull/1325/files#diff-f8e049a1b6d4f9aa96fca41d2f7aa11dL697

I don't know _why_ it did that check, but it doesn't look like this pull-request changes any functionality.